README.md

```diff
@@ -160,20 +150,14 @@ Datawow.setup do |config|
 end
 ```
 
-After configuration has been set. You can calling APIs by not insert token like this
-```ruby
-Datawow.[class].create({data: "image URL"})
-```
-The system will be looking for your token by automatically
-
 # Response
 
 Response is a module and it contain such as meta-data, message, status and data.
 #### Example of response module
 ```ruby
 <Datawow::Response @status=200, @message="success" @meta={"code"=>200, "message"=>"success"}, @data={...}, />
 ```
-you can use data whit calling `data` property and you will be get data like a example below
+you can use data with call `data` property and you will be get data like a example below
```

Comment:
You can call the `data` property to get the data, which would return in the format as shown below